google/cloud/datastore/query.py

```diff
@@ -697,21 +897,6 @@ def _pb_from_query(query):
         ancestor_filter.op = query_pb2.PropertyFilter.Operator.HAS_ANCESTOR
         ancestor_filter.value.key_value.CopyFrom(ancestor_pb._pb)
 
-    for property_name, operator, value in query.filters:
-        pb_op_enum = query.OPERATORS.get(operator)
-
-        # Add the specific filter
-        property_filter = composite_filter.filters._pb.add().property_filter
-        property_filter.property.name = property_name
-        property_filter.op = pb_op_enum
-
-        # Set the value to filter on based on the type.
-        if property_name == "__key__":
```

Comment:
how come this logic is being removed?

Reply:
This is moved the the `PropertyFilter` constructor, which is a new datastructure that is meant as a replacement of the `property_name`, `operator`, and `value` variables.
See https://github.com/googleapis/python-datastore/pull/418/files/a6cfacfdd674de0cf59fd6571c05ae4d1624318e#diff-6053552b52329f41530b743e3fcb37f9e7622c7e165f751b94d08174df79ac26R70 where this gets moved to